javascript/ql/src/Security/CWE-116/BadTagFilter.qhelp

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Parsing general HTML using regular expressions is impossible, however it is possible to match
+single HTML tags. However, if the regexp is not written well it might be easy 
+to circumvent the regexp, which can lead to XSS or other security issues.
```

Comment:
```suggestion
Parsing general HTML using regular expressions is impossible, though it is possible to match
single HTML tags. However, if the regular expression is not written well it might be possible 
to circumvent it, which can lead to cross-site scripting or other security issues.
```